CTP/shell/src/com/navercorp/cubridqa/shell/common/ShellInput.java

```diff
@@ -86,14 +86,10 @@ public String getCommands(String user) {
 		
 		if(!isWindows) {
 			return result;
-		} else if(SSHConnect.SERVICE_PROTOCOL.equals("ssh")){
```

Comment:
Since we don't prepare to support windows shell over ssh, hope to just do 'return result'.